extensions/amp-story/1.0/amp-story-share.js

```diff
@@ -314,7 +326,7 @@ export class ShareWidget {
     this.loadRequiredExtensions();
 
     this.requestService_.loadBookendConfig().then(config => {
-      const providers = config && config[SHARE_PROVIDERS];
+      const providers = config && this.readProviders(config);
```

Comment:
What do you think of simply doing 
````
const providers = config && config[SHARE_PROVIDERS] || config['share-providers'];
````
here, so you don't have to write a bookend error message in the share widget file?
The bookend error message should be moved to the other class, with the bookend version warning.

Reply:
ok, only thing is that this way we won't show the warning when they use the 'share-providers' key. Just making sure we are ok with that